src/Optimizations/peps.jl

```diff
@@ -19,10 +19,10 @@ function loss_grad_wrap(peps::PEPS, Hlocal::Array)
   return loss_w_grad
 end
 
-function loss_grad_wrap(peps::PEPS, Hlocal::Array, ::typeof(insert_projectors))
+function loss_grad_wrap(peps::PEPS, Hlocal::Array, ::typeof(insert_projectors), maxdim=100)
```

Comment:
Also would be good to add a `cutoff` keyword argument which truncates the projector/boundary MPS according the magnitude of the singular values.

Reply:
Done